tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -604,10 +759,17 @@ def do_test(self, test):
         return (test.make_result(*data), [])
 
     def do_testharness(self, protocol, url, timeout):
+        # TODO: respect timeout.
```

Comment:
could you expand on what is exactly needed? it sounds important to implement before this lands.

Reply:
I removed the comment. The `timeout` argument is used in some `TimedRunner` usages (`ServoWebDriverTestharnessExecutor`), but not in all (`WebDriverRun`, `ServoTimedRunner`). This TODO is out of scope of the PR and I'm not sure if relevant at all.